### ISSUE TITLE
add unicode font family and webfonts in gitbook web version

### ISSUE THREAD

**Issue by billryan** [billryan/algorithm-exercise#7]
The font family used in [gitbook web version](http://algorithm.yuanbin.me) are unfriendly to unicode fonts such as Chinese character.

The `font-family` is set by default as

``` css
font-family:"Open Sans","Clear Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
```

which looks ugly on Windows platform. I should use some extra plugin to solve this problem.

简单来讲就是目前的 Gitbook 网页版生成的中文字体显示效果不好，OS X/Linux 下还算可以，Windows 下的渲染效果则惨不忍睹。

另外对于非中文的字体，计划使用 Webfont 来解决，尤其是代码部分，使用`source code pro`等字型将大大提高代码的可读性和观赏性。Reference 中的 yahei 还算凑合，不过我想将它扩充到支持 Windows/Linux/OS X/Android/iOS 等多个平台，而不仅仅只是对 Windows 用户用好。嗯，这样的话就得写 js 插件了，过些天找好实习了就来 fork 一个，有略懂前端的小伙伴愿意协作的咩？
## Reference
- [zhangzhb3/gitbook-plugin-yahei](https://github.com/zhangzhb3/gitbook-plugin-yahei)
- [setkyar/gitbook-plugin-unicodecss](https://github.com/setkyar/gitbook-plugin-unicodecss)
- [sofish/typo.css](https://github.com/sofish/typo.css/tree/master) - 一致化浏览器排版效果，构建最适合中文阅读的网页排版
- [Create a plugin | Gitbook](http://help.gitbook.com/plugins/create.html) - 发布插件用

**Comment by billryan** [comment]
## Website

Fixed in billryan@dd6086c6bc2686e04e504404a21033d8a409d100
`plugin-yahei` and `plugin-unicodecss` are deprecated.

## To-Do

Customize `styles/pdf.css`, `styles/epub.css` and `styles/mobi.css`.

**Comment by billryan** [comment]
- sans-serif: Noto Sans, Noto Sans CJK, Arial(temporary) - 
  [arial.zip](https://github.com/billryan/algorithm-exercise/files/279471/arial.zip)
- monospace: "Migu 1M" 开源字体 -
  [migu-1m-20150712.zip](https://github.com/billryan/algorithm-exercise/files/277714/migu-1m-20150712.zip)
- serif: None
- [Linux字体美化实战(Fontconfig配置) [金步国]](http://www.jinbuguo.com/gui/linux_fontconfig.html)
